benches/fibonacci_lem.rs

```diff
@@ -59,44 +59,73 @@ impl ProveParams {
     fn name(&self) -> String {
         format!("Fibonacci-rc={}", self.reduction_count)
     }
+
+    fn params(&self) -> String {
+        let output_type = bench_parameters_env().unwrap_or("stdout".into());
+        match output_type.as_ref() {
+            "pr-comment" => format!("num-{}", self.fib_n),
+            "commit-comment" => todo!(),
+            "gh-pages" => todo!(),
+            _ => format!("num-{}/{}-{}", self.fib_n, self.sha, self.date),
+        }
+    }
+}
+
+fn bench_parameters_env() -> anyhow::Result<String> {
+    std::env::var("LURK_BENCH_OUTPUT")
+        .map_err(|e| anyhow!("Noise threshold env var isn't set: {e}"))
+}
```

Comment:
At this stage, this probably indicates we want to document what folks are allowed to put in the variables defined in the env file.